python/llm/src/bigdl/llm/transformers/model.py

```diff
@@ -72,6 +72,9 @@ def from_pretrained(cls,
             kwargs["low_cpu_mem_usage"] = True
             # set default torch_dtype='auto'
             kwargs["torch_dtype"] = kwargs.get("torch_dtype", 'auto')
+            # Avoid tensor parallel F.Linear Operations
+            if "pretraining_tp" in config_dict:
+                kwargs["pretraining_tp"] = 0
```

Comment:
should it be 1?

Reply:
I think both 0 and 1 will prevent this problem, only if self.pretraining_tp <= 1
```
        if self.pretraining_tp > 1: 
```

Reply:
what the default value for 7B/13B?

Reply:
They don't have pretraining_tp this entry in their config.json